docs/src/snap/howto/ceph.md

```diff
@@ -0,0 +1,306 @@
+# How to use ceph
+
+Distributed, redundant storage is a must-have when you want to develop reliable applications. Ceph is a storage solution which provides exactly that, and is built with distributed clusters in mind. In this tutorial, we'll be looking at how to integrate Canonical Kubernetes with a Ceph cluster. Specifically, by the end of this tutorial you'll have a Kubernetes pod with a mounted RBD-backed volume. This how to is adapted from [block-devices-and-kubernetes].
+
+## What you'll need
+
+This guide assumes the following:
+
+- You have root or sudo access to the machine
+- You have a bootstrapped Canonical Kubernetes cluster (see the
+  [getting-started-guide])
+- You have a running Ceph cluster
+
+## Create a Ceph storage pool
+
+Create a storage pool named "kubernetes" in the Ceph cluster.
+
+We will set the number of placement groups to 128 because the Ceph cluster of this demonstration will have less than 5 OSDs. (See [placement groups])
```

Comment:
These are very specific Ceph terms. I'm not sure if we need to briefly explain them here or if we just assume familiarity with Ceph for this guide, @evilnick ?

Reply:
It probably isn't necessary in a how to. We have assumed they have deployed Ceph and there are a few links to explanitory upstream docs. But yeah, lets not go overboard either.